pgsmo/objects/functions/procedure.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do we want to have a separate folder for procedures?

Reply:
I think staying here is fine, since it uses the same base as function and trigger function